### PR TITLE
Codename: Jeu Tarot- Tarot Cards, for games and divination

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request

Adds tarot cards, for business and pleasure. The tarot deck consists of 78 cards, split into 56 Minor Arcana cards (each suit consists of 10 number cards, and 4 face cards) and 22 Major Arcana cards. The deck has new art:
minorarcana
majorarcana
Special thanks to Fikou for deciding which jobs should correspond to each of the Major Arcana.
## Why It's Good For The Game
It's a fun distraction from the dull humdrum of station life, and opens up some good RP opportunities.

## Changelog
:cl: Inept
add: The Nanotrasen Entertainment Division has released a new product: Tarot cards! They're available from the Fun vendor for all your divination needs.
/:cl:

### CODE REVIEW

**Comment by spessbandit** [comment]
The Fool should be a clown

**Comment by HomocidlePancakes** [comment]
*The moment man devoured the fruit of knowledge, he sealed his fate... Entrusting his future to the cards, man clings to a dim hope. Yes, the Arcana is the means by which all is revealed...*

File: code/modules/games/tarot.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		for(var/person in list("Valet", "Chevalier", "Dame", "Roi"))
+			cards += "[person] of [suit]"
+	for(var/trump in list("The Magician", "The High Priestess", "The Empress", "The Emperor", "The Hierophant", "The Lover", "The Chariot", "Justice", "The Hermit", "The Wheel of Fortune", "Strength", "The Hanged Man", "Death", "Temperance", "The Devil", "The Tower", "The Star", "The Moon", "The Sun", "Judgement", "The World", "The Fool"))
+		cards += "[trump]"
+
+/obj/item/toy/cards/deck/tarot/draw_card(mob/user)
+	. = ..()
+	var/obj/item/toy/cards/singlecard/C = .
+	var/matrix/M = matrix()
+	M.Turn(180)
```
Comment (lordpidey) [inline code/modules/games/tarot.dm:22]: Move the matrix creation and rotation to inside of the if statement.  No sense in creating and manipulating a matrix if it won't be used.

**Review by lordpidey — CHANGES_REQUESTED** [review]

**Comment by Tharcoonvagh** [comment]
change the deck sprite to the holopara tarot cards'

**Review by kriskog — DISMISSED** [review]
.

**Comment by EOBGames** [comment]
Oh no, this PR has drawn the Tower!

**Pull request closed without merging**